src/ui/czi-ordered-list-counters.scss

```diff
@@ -0,0 +1,12 @@
+// prettier-ignore
+
+// This generate many rules for resetiing the counter list with specific
+// start number. Ideally we could just use one line of CSS like
+// `.ProseMirror ol[start] { counter-reset: czi-counter attr(start); }
+// but unfortunately no browser ever support it so we could only use SASS to
+// do it.
+@for $i from 1 to 500 {
+  .ProseMirror ol[start='#{$i}'] {
```

Comment:
The gzipped size is about 2k for the generated CSS.
I'm hesitant to use JS cause the CSS is less readable in JS but I'm open to try that if SASS isn't really an option.

Reply:
JS is built for this, this is what inline styles are for.

Reply:
I'm not sure what you meant by "inline styles". Did you propose using a JS library to do it or using vanilla JS  like this?

```
const el = document.createElement('style');
el.textContext = ..... // 
document.head.appendChild(el);
```

What I don't like about this approach (vanilla JS) for these reasons:

- The CSS rules are less readable because they are treated as JS strings.
- I need to find a component and inject the styles safely when the DOM is ready, which seems uglier than `import 'mycss.css'`.